sdk/tests/conformance/resources/gl-object-get-calls.js

```diff
@@ -122,6 +121,11 @@ gl.bindTexture(gl.TEXTURE_2D, null);
 var framebuffer = gl.createFramebuffer();
 gl.bindFramebuffer(gl.FRAMEBUFFER, framebuffer);
 gl.framebufferTexture2D(gl.FRAMEBUFFER, gl.COLOR_ATTACHMENT0, gl.TEXTURE_2D, texture, 0);
+var colorAttachmentsNum = 1;
+if (contextVersion > 1) {
+  colorAttachmentsNum = gl.getParameter(gl.MAX_COLOR_ATTACHMENTS);
+  gl.framebufferTexture2D(gl.FRAMEBUFFER, gl.COLOR_ATTACHMENT0 + colorAttachmentsNum - 1, gl.TEXTURE_2D, texture, 0);
```

Comment:
OK, I see. Sorry for my confusion. I didn't read the loop header carefully enough. It isn't very easy to read. Please try to keep your code simple and easy to understand.